test/v5/bfLoss/ListBFLossesFixtures.scala

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v5.bfLoss
+
+import api.models.domain.TaxYear
+import play.api.libs.json.{JsValue, Json}
+import v5.bfLosses.list.def1.model.response.{Def1_ListBFLossesItem, Def1_ListBFLossesResponse}
+import v5.bfLosses.list.model._
+
+object ListBFLossesFixtures {
```

Comment:
This is likely going to be schema specific so should probably be `Def1_ListBFLossesFixtures` under the def1 package.